spec/factories/vm_azure.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :vm_azure, :class => "ManageIQ::Providers::Azure::CloudManager::Vm", :parent => :vm_cloud do
     location "westus"
-    vendor   "azure"
+    vendor   "microsoft"
```

Comment:
@bzwei I think "azure" was correct based on the `VENDOR_TYPES` defined in [vm_or_templates.rb](https://github.com/ManageIQ/manageiq/blob/master/app/models/vm_or_template.rb#L36).

@blomquisg @djberg96 @bronaghs Please verify.

If it should be "azure" that effects the updates to the `ManageIQ/Cloud/VM/Retirement/StateMachines/Methods` schema.